starter-code/src/App.js

```diff
@@ -1,18 +1,92 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
+import contacts from './contacts.json';
+import ContactsTable from './components/ContactsTable';
+import SmallButton from './components/SmallButton';
 import './App.css';
 
+const NUM_CONTACTS_TO_IMPORT = 5;
+
 class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      ironContacts: contacts.filter((contact, index) => {
+        if (index < NUM_CONTACTS_TO_IMPORT) {
+          contact.index = index;
+          return contact;
+        }
+        return null;
+      }),
+    }
+  }
+
+  selectRandom = (range) => {
+    return Math.floor((Math.random() * range));
+  }
```

Comment:
ya q estás escribiendo buen código, por lo q veo, si no vas a meter lógica en una función, y sólo vas a retornar algo sencillito como este math, podrías

`selectRandom = (range) => Math.floor((Math.random() * range))`

y quedas como the king!

Reply:
gracias Alex!